routes/api.php

```diff
@@ -83,5 +83,6 @@
         Route::get('/admin/nested-categories/{subcategory_id}', [CategoryController::class, 'getSubcategoryParentCategories']);
         Route::post('/admin/profile-edit', [ChangeNameEmailController::class, 'restore']);
         Route::patch('/admin/password-edit', [AdminController::class, 'updatePassword']);
+        Route::get('/admin/admins', [AdminController::class, 'getAdminAccounts']);
```

Comment:
admin/admins sounds too repetitive. please update to another name
```suggestion
        Route::get('/admin/users', [AdminController::class, 'getAdminAccounts']);
```

is /admin/users not used yet?